source/_components/group.markdown

```diff
@@ -89,3 +89,34 @@ Notice in the example below that in order to refer to the group "Living Room", y
       - group.living_room
       - group.bedroom
 ``` 
+
+## {% linkable_title Default groups %}
+
+Some components automatically creates special groups, which contains component entities. These groups are named like `group.all_...`, for example:
```

Comment:
I think the first sentence would be a bit clearer written as:
"Some components automatically create special groups containing component entities."